<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCLViewer</class>
 <widget class="QMainWindow" name="PCLViewer">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>498</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PCLViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKWidget" name="qvtkWidget" native="true">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>9</y>
      <width>661</width>
      <height>480</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>50</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_ColorMode">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>236</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>236</width>
      <height>94</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <stylestrategy>NoAntialias</stylestrategy>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(128, 255, 245);</string>
    </property>
    <property name="title">
     <string>Color mode</string>
    </property>
    <widget class="QRadioButton" name="radioButton_BlueRed">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Blue to red</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_WhiteRed">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>156</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>White to red</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_GreyRed">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Grey / red</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_GreenMagenta">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>218</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Green to magenta</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Rainbow">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>115</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Rainbow</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_x">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>40</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_y">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>38</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_z">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>40</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Save file</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_icp">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 224, 128);</string>
    </property>
    <property name="title">
     <string>set iteration </string>
    </property>
    <widget class="QSpinBox" name="spinBoxCount">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(247, 255, 255);</string>
     </property>
     <property name="minimum">
      <number>30</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>20</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_load">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 17, 92);</string>
    </property>
    <property name="text">
     <string>Load file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open_right">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>open_model</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(44, 54, 255);</string>
    </property>
    <property name="text">
     <string>&lt;h1&gt;Deng DX GUI &lt;/h1&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open_left">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 97, 255);</string>
    </property>
    <property name="text">
     <string>open_icp_file</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
